dashboard/models/tile.py

```diff
@@ -78,42 +74,23 @@ def _touch(self):
             fhandle.close()
 
     def sync_config(self):
-        self.connect_s3()
+        """Determines if the config file is updated and if so fetches the new config."""
         try:
+            logger.info("Config file is updated fetching new config.")
+            self._get_config()
```

Comment:
Shouldn't this stay in the `if self.is_updated():` block?  otherwise it pull the apps.yml on every request which we don't want.